src/java/map/ConcurrentFloatMask.java

```diff
@@ -238,6 +238,11 @@ public void show() {
         this.floatMask.show();
     }
 
+    public ConcurrentFloatMask startVisualDebugger() {
+        this.floatMask.startVisualDebugger(Util.getStackTraceParentClass());
+        return this;
+    }
+
```

Comment:
Why was this added masks need a name in the debugger?

Reply:
it uses the toString() name from the FloatMask function I made...